### PR TITLE
Update TestX86.java

### PR DESCRIPTION
**Pull request by keenk**
Added printouts for the encoding class
Ported code from cstool to print registry access modes

### CODE REVIEW

File: bindings/java/TestX86.java [hunk 1]
```diff
@@ -131,6 +145,40 @@ public static void print_ins_detail(Capstone.CsInsn ins) {
...
+            break;
+          case CS_AC_READ | CS_AC_WRITE:
+            System.out.printf("\t\toperands[%d].access: READ | WRITE\n", c);
+            break;
+        }
+      }
+
+      // Print out all registers accessed by this instruction (either implicit or explicit)
+      CsRegsAccess regsAccess = ins.regsAccess();
+      if(regsAccess != null) {
```
Comment (aquynh) [inline bindings/java/TestX86.java]: please put a space between `if` and `(`

**Comment by aquynh** [comment]
great, but can we print those encoding fields only when they are relevant?
see https://github.com/aquynh/capstone/blob/next/tests/test_x86.c

**Comment by keenk** [comment]
Added checks and spaces

File: bindings/java/TestX86.java [hunk 2]
```diff
@@ -58,9 +61,24 @@ public static void print_ins_detail(Capstone.CsInsn ins) {
...
     // print displacement value
     System.out.printf("\tdisp: 0x%x\n", operands.disp);
 
+    // print displacement offset
+    if (operands.encoding.dispOffset != 0) {
+      System.out.printf("\tdisp offset: 0x%x\n", operands.encoding.dispOffset);
+    }
+
+    //print displacement size
+    if(operands.encoding.dispSize != 0) {
```
Comment (aquynh) [inline bindings/java/TestX86.java]: a space here, too.

**Comment by keenk** [comment]
Space added.

What about:
https://github.com/keenk/capstone/blob/a1b1cf73c8b875a6ab92e9bad45c9a6e42f1bfb0/bindings/java/TestX86.java#L154

I thought about adding a space there as well, but in test_x86.c the switches don't have a space.

https://github.com/keenk/capstone/blob/a1b1cf73c8b875a6ab92e9bad45c9a6e42f1bfb0/tests/test_x86.c#L36

https://github.com/keenk/capstone/blob/a1b1cf73c8b875a6ab92e9bad45c9a6e42f1bfb0/tests/test_x86.c#L228

https://github.com/keenk/capstone/blob/a1b1cf73c8b875a6ab92e9bad45c9a6e42f1bfb0/tests/test_x86.c#L262

**Comment by aquynh** [comment]
merged, thanks!

yes, switch is inconsistent, it is my habit to leave no space there. feel free to fix them with pull reqs.

**Pull request merged**